charts/consul/values.yaml

```diff
@@ -1859,6 +1859,85 @@ connectInject:
     # Note: This value has no effect if transparent proxy is disabled on the pod.
     defaultOverwriteProbes: true
 
+  # Configures consul-cni plugin for Consul Service mesh services
+  # Using this feature requires Consul 1.13.0-beta+
+  cni:
+    # If true, then all traffic redirection setup with use the consul-cni plugin
+    # @type: boolean
+    enabled: false
+
+    # Namespace to install cni plugin.
+    # @type: string
+    namespace: kube-system
+ 
+    # If multus plugin is enabled or not. If true, the cni installer will generate a consul cni configuration
+    # in a location for the multus program to use. Also, the helm chart will create a custom resource for
+    # NetworkAttachmentDefintion that multis. cniBinDir and cniNetDir must be changed to work with multus.
+    # Please refer to your cloud platform for more informaton on using multus.
```

Comment:
Are there any official docs for multus we can link to?

Reply:
There is very little that I can find. There is the official "how to install multus on your kubernetes cluster" docs and at least 1 for Istio.  I think that we will need a learn guide for using Multus (on OpenShift most likely) and I want to write that post CNI. We have at least 1 customer that could use a doc for it.

Reply:
Ah ok, got it!